config/site_settings.yml

```diff
@@ -361,6 +361,10 @@ basic:
     default: true
   sitemap_page_size:
     default: 10000
+  enable_user_status:
+    hidden: true
```

Comment:
I assume the hidden is on purpose for now?

Reply:
Yep, this is the very first iteration, we're going to add more functionality to it soon.